test/tst.layout.js.out

```diff
@@ -280,12 +280,12 @@ summary:
      loadbalancer         -                3
      ops                  -                1
      madtom               -                1
-     reshard              -                 
-     pgstatsmon           -                 
-     garbage-collector     -                 
-     prometheus           -                 
+     reshard              -
+     pgstatsmon           -
+     garbage-collector     -
+     prometheus           -
```

Comment:
FWIW, this might cause the tests to fail. The table-printing lib that manta-adm uses (node-tab) will emit trailing whitespace, unfortunately.